src/ctx.cpp

```diff
@@ -1364,11 +1364,12 @@ llvm::Value *FunctionEmitContext::MasksAllEqual(llvm::Value *v1, llvm::Value *v2
 #else
     if (g->target->getArch() == Arch::wasm32) {
         llvm::Function *fmm = m->module->getFunction("__wasm_cmp_msk_eq");
-        return CallInst(fmm, NULL, {v1, v2}, LLVMGetName("wasm_cmp_msk_eq", v1, v2));
+        return CallInst(fmm, NULL, {v1, v2}, llvm::Twine("wasm_cmp_msk_eq_") + v1->getName() + "_" + v2->getName());
```

Comment:
This seems to be dependent on the order of expression evaluation, which is not guaranteed.
I.e. if evaluated from right to left, it will yield an error.
Adding extra braces for ordering expressions seems to be ok-ish solution here.

Reply:
Done